src/builder.spec.ts

```diff
@@ -1,5 +1,46 @@
+import { Builder } from './builder';
+
 describe('Builder', () => {
-  it('works', () => {
-    expect(true).toBe(true);
+  const processCwd = jest.spyOn(process, 'cwd').mockReturnValue('/home/johndoe/my-report');
+
+  const loadPackageJson = jest.fn();
+  const rimraf = jest.fn().mockResolvedValue(undefined);
+  const exec = jest.fn().mockResolvedValue({ stdout: 'stdout', stderr: 'stderr' });
```

Comment:
I guess I was right about the mocking :)

Reply:
You could also mock the dependencies of `Builder`, independently on how `Builder` uses them in any form. Something like:

```ts
import * as fileHelpers from './file.helpers';

jest.mock('./file.helpers', () => {
  loadPackageJson: jest.fn(),
  ...fileHelpers
});

...

fileHelpers.loadPackageJson.mockReturnValueOnce({});
```

I didn't fully check if this is working, but it's how jest allows you to mock other modules. Then, it's unimportant how `Builder` uses its imported modules.